tests/run-centos-ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+pip install virtualenv
```

Comment:
This is wrong

* Do not install virtualenv with pip
* You won't actually have pip at this point because the python-virtualenv package is what installs it.

Move the yum instal lines from 11-12 here. Combine them into one line. Remove this one.